x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/components/csp_details/alerts_findings_details_table.tsx

```diff
@@ -124,7 +145,9 @@ export const AlertsDetailsTable = memo(
       color: getSeverityColor(key),
       filter: () => {
         setCurrentFilter(key);
-        setQuery(buildEntityAlertsQuery(field, to, from, value, 500, key));
+        setQuery(
+          buildEntityAlertsQuery(field, to, from, value, 500, key, sortField, sortDirection)
```

Comment:
usually with this many fields, its better to use an object, consider a quick refactor for `buildEntityAlertsQuery`